libs/shared/src/types/storage/index.ts

```diff
@@ -0,0 +1,11 @@
+export enum MimeTypesEnum {
+  JPEG = 'jpeg',
+  PNG = 'png',
+  JPG = 'jpg',
+}
+
+export const MIME_TYPES_LOOKUP: Record<MimeTypesEnum, string> = {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice!‏